docs/src/components/Demo/index.js

```diff
@@ -65,7 +65,7 @@ const Demo = ({ classes }) => {
   }, [])
 
   const showCroppedImage = useCallback(async () => {
-    const croppedImage = await getCroppedImg(dogImg, croppedAreaPixels)
+    const croppedImage = await getCroppedImg(dogImg, croppedAreaPixels, rotation)
     setCroppedImage(croppedImage)
   }, [croppedAreaPixels])
```

Comment:
You need to add the `rotation` to the deps array of `useCallback()`. Otherwise, if you change the rotation twice in a row, the output wont't have the right rotation.

Reply:
Good spot, I’m so used to have the`rule-of-hooks` linter autofix that for me I never think to check.

Reply:
Yes, same here. 😄 We need to add this to this project.